docker-compose.yml

```diff
@@ -51,6 +51,12 @@ services:
       CROWN_API_SECRET: ${CROWN_API_SECRET:? CROWN_API_SECRET not set}
       CROWN_WITHDRAW_FEE_INT: ${CROWN_WITHDRAW_FEE_INT:? CROWN_WITHDRAW_FEE_INT not set}
       CROWN_WITHDRAW_NAME: ${CROWN_WITHDRAW_NAME:? CROWN_WITHDRAW_NAME not set}
+      THRESHOLD_TOTAL_SPENDABLE: ${THRESHOLD_TOTAL_SPENDABLE:-1000000}
+      THRESHOLD_TOTAL_RECEIVABLE:  ${THRESHOLD_TOTAL_RECEIVABLE:-1000000}
+      THRESHOLD_TOTAL_ONCHAIN_BAL: ${THRESHOLD_TOTAL_ONCHAIN_BAL:-3000000}
+      THRESHOLD_AVAIL_BTC_EXCH_BAL: ${THRESHOLD_AVAIL_BTC_EXCH_BAL:-0.05}
+      THRESHOLD_AVAIL_NZD_EXCH_BAL: ${THRESHOLD_AVAIL_NZD_EXCH_BAL:-2000}
+      THRESHOLD_AVAIL_NZD_BANK_BAL: ${THRESHOLD_AVAIL_NZD_BANK_BAL:-2000}
```

Comment:
```suggestion
```

you dont actually need to put this here for .env variables to get passed to the container

Reply:
> ill need help, not sure how to fix the following issue:
> 
> ```
>   /home/runner/work/beryllium/beryllium/src/app_core.py:126:47 - error: Argument of type "str | None" cannot be assigned to parameter "__x" of type "ReadableBuffer | str | SupportsInt | SupportsIndex | SupportsTrunc" in function "__new__"
>     Type "str | None" cannot be assigned to type "ReadableBuffer | str | SupportsInt | SupportsIndex | SupportsTrunc"
>       Type "None" cannot be assigned to type "ReadableBuffer | str | SupportsInt | SupportsIndex | SupportsTrunc"
>         Type "None" cannot be assigned to type "str"
>         Type "None" cannot be assigned to type "ReadOnlyBuffer"
>         Type "None" cannot be assigned to type "bytearray"
>         Type "None" cannot be assigned to type "memoryview"
>         Type "None" cannot be assigned to type "array[Any]"
>         Type "None" cannot be assigned to type "mmap"
>     ... (reportGeneralTypeIssues)
>   /home/runner/work/beryllium/beryllium/src/app_core.py:127:48 - error: Argument of type "str | None" cannot be assigned to parameter "__x" of type "ReadableBuffer | str | SupportsInt | SupportsIndex | SupportsTrunc" in function "__new__"
>     Type "str | None" cannot be assigned to type "ReadableBuffer | str | SupportsInt | SupportsIndex | SupportsTrunc"
>       Type "None" cannot be assigned to type "ReadableBuffer | str | SupportsInt | SupportsIndex | SupportsTrunc"
>         Type "None" cannot be assigned to type "str"
>         Type "None" cannot be assigned to type "ReadOnlyBuffer"
>         Type "None" cannot be assigned to type "bytearray"
>         Type "None" cannot be assigned to type "memoryview"
>         Type "None" cannot be assigned to type "array[Any]"
>         Type "None" cannot be assigned to type "mmap"
>     ... (reportGeneralTypeIssues)
> ```
> 
> i think the above issue is caused by the value being set to None even though a default value is configured for variables not set in the env file. will try to push again but this time i want it to not start when the variables are not set. and to continue when the variables are set. wanted to see if the actions error message disappears when the its configured this way.

it might be because there was no default param to the os.getenv function

Reply:
making the changes.